packages/shared/client/ReplayClient.ts

```diff
@@ -261,24 +259,35 @@ export class ReplayClient implements ReplayClientInterface {
   }
 
   async findAnnotations(kind: string, listener: AnnotationListener) {
-    const sessionId = this.getSessionIdThrows();
+    const sessionId = await this.waitForSession();
     assert(!this.annotationListeners.has(kind), `Annotations of kind ${kind} requested twice`);
     this.annotationListeners.set(kind, listener);
     await client.Session.findAnnotations({ kind }, sessionId);
   }
 
-  async findKeyboardEvents(onKeyboardEvents: (events: keyboardEvents) => void) {
-    const sessionId = this.getSessionIdThrows();
+  async findKeyboardEvents() {
```

Comment:
`findKeyboardEvents ` and `findNavigationEvents` accepted a callback so they could be used as streaming APIs, but nothing took advantage of this because those points come in quickly, so this PR also simplified them to be simpler to call.